### ISSUE TITLE
Reverse GeoCoding

### ISSUE THREAD

**Issue by UsamaKarim** [liodali/osm_flutter#84]
It will also be helpful if we get the address of the location (not only GeoPoint) if we select the marker while picking the location. I know that it can be achieved without any service and some Packages like [GeoCoding](https://pub.dev/packages/geocoding) have the feature to get the address from Platform's own services. But the users of your package will be mostly from the countries like China where Google Play Services do not work

**Comment by jesussmile** [comment]
Yes! I was thinking the same, currently with this plugin is there a way to get address from location, reverse geocoding?

**Comment by liodali** [comment]
I will add static method  to reverse geopoint and inverse also
i will use nominatim api
it will available innex week inshalah

**Comment by jesussmile** [comment]
> I will add static method to reverse geopoint and inverse also
> i will use nominatim api
> it will available innex week inshalah

https://github.com/liodali/osm_flutter/discussions/86

**Comment by liodali** [comment]
@UsamaKarim 
you can use this library it implement what i want to add to this library
https://pub.dev/packages/osm_nominatim
